### PR TITLE
Remove faq/applications

### PR DESCRIPTION
**Pull request by howardjohn**
This documentation was all about working around networking which are all
resolved automatically by
https://istio.io/latest/blog/2021/upcoming-networking-changes/

### CODE REVIEW

**Review by linsun — APPROVED** [review]
Thanks! I was going to remove it :)

**Comment by linsun** [comment]
+1 I like that @craigbox

File: content/en/about/faq/traffic-management/statefulsets.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: Can Istio be used with StatefulSets and headless Services?
```
Comment (ericvn) [inline content/en/about/faq/traffic-management/statefulsets.md:2]: I think the capitalization of headless should match services. I'm in favor of uppercasing the `h`.

Comment (howardjohn) [reply content/en/about/faq/traffic-management/statefulsets.md:2]: From k8s docs: "In this case, you can create what are termed "headless" Services". "headless" is not a proper noun while Services and StatefulSets are referring to the object names?

Comment (ericvn) [reply content/en/about/faq/traffic-management/statefulsets.md:2]: Interesting as I found k8s docs like, https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/, that have `StatefulSets currently require a Headless Service to be responsible...`

I'll take your comment as a guide.

File: content/en/about/faq/traffic-management/statefulsets.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+weight: 30
```
Comment (ericvn) [inline content/en/about/faq/traffic-management/statefulsets.md:3]: Can we update this to 25 or 35, just so we aren't overloading 30?

Comment (howardjohn) [reply content/en/about/faq/traffic-management/statefulsets.md:3]: updated

**Review by ericvn — APPROVED** [review]

**Pull request merged**